test/planners.jl

```diff
@@ -19,6 +19,17 @@ sol = planner(blocksworld, bw_state, bw_spec)
 @test is_goal(bw_spec, blocksworld, sol.trajectory[end])
 @test collect(sol) == @pddl("(pick-up a)", "(stack a b)",
                             "(pick-up c)", "(stack c a)")
+sol = planner(wgc_domain, wgc_state, wgc_spec)
```

Comment:
An alternative to copy pasting this code for each planner would be a loop over the planners.

Reply:
Yup, at some point I might do that when planner coverage across all forms of functionality is higher! For now it's a bit more complicated because some planners don't support certain kinds of specifications.